### ISSUE TITLE
Current worksheet no longer synchronize across GNU screens after detaching

### ISSUE THREAD

**Issue by ppasupat** [codalab/codalab-worksheets#168]
When using GNU screen, changing the current worksheet in one screen will affect other screens as well:

```
> screen
> cl work          => current worksheet = local::codalab
> cl work hello    => current worksheet = hello
[create a new screen by pressing ^A c]
> cl work          => current worksheet = hello
> cl work world    => current worksheet = world
[go back to previous screen by pressing ^A p]
> cl work          => current worksheet = world
```

However, after detaching and reattaching the screen, the current worksheet no longer sync up:

```
[detach by pressing ^A d]
> screen -r        # reattach screen
> cl work          => current worksheet is reverted back to local::codalab
> cl work hello    => current worksheet = hello
[go to the next screen (#1) by pressing ^A n]
> cl work          => current worksheet = local::codalab instead of hello
> cl work hello    => current worksheet = hello
[create a new screen by pressing ^A c]
> cl work          => current worksheet = local::codalab instead of hello
```

**Comment by percyliang** [comment]
This was supposed to be a feature, so you don't have to switch sessions that often, but I can see how it might be very confusing.  Will fix.

**Comment by percyliang** [comment]
Fixed by #254
